instrumentation/dalli/lib/opentelemetry/instrumentation/dalli/patches/server.rb

```diff
@@ -17,7 +17,7 @@ def request(op, *args) # rubocop:disable Naming/MethodParameterName
               'db.operation' => operation,
               'net.peer.name' => hostname,
               'net.peer.port' => port
-            }
+            }.compact
```

Comment:
`.compact!` means one less hash allocation, it can return `nil` so don't call it on the closing `}`. 
```suggestion
            }
            attributes.compact!
```